ui-poc/pages/Friends.py

```
@@ -0,0 +1,350 @@
+        response = make_api_request(
+            'get',
+            f"{API_URL}/groups",
+            headers=headers
+        )
+        
+        if st.session_state.debug_mode:
+            st.sidebar.subheader("Debug: /groups response")
+            st.sidebar.write(f"Status Code: {response.status_code}")
+            try:
+                st.sidebar.json(response.json())
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)